### ISSUE TITLE
Round menu idea - a feature proposition

### ISSUE THREAD

**Issue by v6itja** [linuxmint/Cinnamon#5406]
Some days ago it just popped into my mind, that why not go for a round-type menus in LM? What I mean is that, when you right-click on something on your Desktop, you get a menu, right? But it has always been squared, where the list goes from up to down. But why not go for a round menu, where you get a color-highlighted option, like copy or paste or whatever? An example would be something like this - https://d13yacurqjgara.cloudfront.net/users/4864/screenshots/696455/circular_menu.jpg
Of course it would be better to have the texts "copy", "open as root" etc written in the menu still, but an icon about a function might or might not be added aswell. But the colors would also be nice - for example "Cut" might be orange-coloured, "Copy" blue, "Create new Folder" green or so.

AND - you would get also the possibility to choose in the System Settings, whether you like round menu or classic/regular type.

Basically, what this rounded menu would give to the user is something new and it might be easier to use. Also more fanciness :D And LM would win by being the first, who implements such a thing in an desktop OS :)

**Comment by PiMaker** [comment]
That would look amazing, but what about dynamically adding options? And options that dont have icons? I think the idea is good but one would definetely have to re- and define it a lot.

**Comment by v6itja** [comment]
By "dynamically adding options" you mean a second-level-menus (for example Create a file -> a text document) ? An outer ring might be considered. And actually I believe that icons are not an inevitable necessity.
But hey, thanks for considering it. At least a new idea to try out and there is also the possibility to receive the users' feedback later, when they have gotten to try it out. :)

**Comment by ManIVIctorious** [comment]
The right click menu you are referring to is handled by nemo (repositiory [linuxmint/nemo](https://github.com/linuxmint/nemo)) Could you please close the issue here and reopen it in the proposed place?

**Comment by JosephMcc** [comment]
@ManIVIctorious is correct. This does belong on the nemo repo.
